### PR TITLE
IMB-129: Resolve internal server error for CEPR/DOB validation

### PR DESCRIPTION
**Pull request by mislam987**
## What?
Resolve internal server error for CEPR/DOB validation [IMB-129](https://collaboration.homeoffice.gov.uk/jira/browse/IMB-129)

## Why?
So that the 'Details not found' screen is shown if a particular CEPR/DOB combination does not match a record in the database (links to [IMB-87](https://collaboration.homeoffice.gov.uk/jira/browse/IMB-87))

## How?

- Add check on `validate-case-details.js` to set the 'duty-to-remove-alert' value in the session only if a record exists in the database for the given CEPR/DOB combination
- Remove line for setting the 'duty-to-remove-alert' form value in `validate-case-details.js`

## Testing?
Tests passing locally

### CODE REVIEW

File: apps/verify/behaviours/validate-case-details.js [hunk 1]
```diff
@@ -15,8 +16,10 @@ module.exports = superclass => class extends superclass {
 
     const imaResult = casesJson.find(
       obj => obj.cepr === cepr && obj['date-of-birth'] === dob);
-    req.form.values['duty-to-remove-alert'] = imaResult['duty-to-remove-alert'];
-    req.sessionModel.set('duty-to-remove-alert', imaResult['duty-to-remove-alert']);
+    if(_.isEmpty(imaResult) === false) {
+      req.form.values['duty-to-remove-alert'] = imaResult['duty-to-remove-alert'];
+      req.sessionModel.set('duty-to-remove-alert', imaResult['duty-to-remove-alert']);
```
Comment (jamiecarterHO) [inline apps/verify/behaviours/validate-case-details.js:21]: Do you need to do both of these?

Comment (mislam987) [reply apps/verify/behaviours/validate-case-details.js]: Tested without the first one that sets the form values and it works. Originally had it in there to test that the `duty-to-remove-alert` was being set properly for it to be picked up in the form later.

Comment (jamiecarterHO) [reply apps/verify/behaviours/validate-case-details.js]: Thanks for updating

**Review by jamiecarterHO — APPROVED** [review]
LgtmM

**Pull request merged**